app/src/routes/+server.js

```diff
@@ -5,15 +5,12 @@ import { json } from '@sveltejs/kit'
 export async function GET({ locals: { db } }) {
 	// https://www.sqlite.org/lang_expr.html#the_like_glob_regexp_match_and_extract_operators
 	const sql = `
-		SELECT name
-		FROM sqlite_master
-		WHERE type = 'table'
-			AND name NOT LIKE '!_%' ESCAPE '!' -- sometimes, CF puts tables in the database, e.g., _cf_KV
-		ORDER BY name
+		SELECT DISTINCT language
```

Comment:
While this is fine for now, we should consider renaming this to something like `target_project`, `project`, etc. A target project is _usually_ a separate language, but not necessarily. For example, the Analyzer project is separate from the English, even though they both use the English _language_.

Reply:
✅  done in [78bb080](https://github.com/presciencelabs/tabitha-targets/commit/78bb080caf06b56ae3e0980fb5a11e2a1548df2a)